assets/tests/TaskSchemaValidationServiceSpec.js

```diff
@@ -44,17 +45,19 @@ describe('TaskSchemaValidationService', function() {
 
   describe('validateAllTasks', function() {
     it('should validate the structure of all sample tasks', function() {
+      var functions = Object.keys(Tsvs);
+      expect(functions.length).toEqual(NUM_FUNCTIONS);
```

Comment:
Really this should be counting just the number of test functions, so start at 0 and increment it in the "if" check, and do the "total number" validation at the end. The benefit to that is that it counts how many tests were actually run, too.